django/http/multipartparser.py

```diff
@@ -580,7 +580,7 @@ def exhaust(stream_or_iterable):
     if iterator is None:
         raise MultiPartParserError('multipartparser.exhaust() was passed a non-iterable or stream parameter')
 
-    for __ in iterator:
+    for _ in iterator:
```

Comment:
I have looked through Django codebase for usages of `_` and found many places where it is used as an unused variable. So I retract my comment.
I guess a conflict is unlikely in well tested code, which Django is.